moped-editor/src/components/GridTable/SearchBar.js

```diff
@@ -206,6 +208,11 @@ const SearchBar = ({
           ),
           endAdornment: (
             <InputAdornment position="end">
+              {!!loading && (
+                <IconButton>
+                  <CircularProgress size="2rem" />
+                </IconButton>
+              )}
```

Comment:
i love the way this looks! i noticed that the spinner is a clickable button, which i'd like to remove but also isn't critical. 

i think the trick to get the spacing right is to add a separate InputAdornment for the spinner. i will open a follow-up issue!

Reply:
thanks, John! I did not mean to make that a clickable button :0 i'll merge now to get this into the release and make these changes in a follow-up pr